test/e2e/framework/providers/aws/aws.go

```diff
@@ -38,7 +38,7 @@ func init() {
 
 func newProvider() (framework.ProviderInterface, error) {
 	if framework.TestContext.CloudConfig.Zone == "" {
-		return nil, fmt.Errorf("gce-zone must be specified for AWS")
+		framework.Logf("Warning: gce-zone not specified! Some tests that use the AWS SDK may select the wrong region and fail.")
```

Comment:
I'll defer to @nckturner on this one. Reference to gce-zone seems confusing in context of the AWS cloud provider. Is this referring to something other than Google Compute Engine?

Reply:
Its just the argument name, I believe `--gce-zone` configures `cloudconfig.zone` so it was just re-used by the aws framework.